.travis.yml

```diff
@@ -144,3 +144,9 @@ matrix:
         - sleep 5
       script:
         - CLICKHOUSE_AVAILABLE=true mvn -Dtest=ClickHouseBinaryComparisonOperationTest test
+    - name: OceanBase
```

Comment:
We are now actually using GitHub Actions for testing. For example, see https://github.com/sqlancer/sqlancer/blob/master/.github/workflows/main.yml#L187 for how the MySQL tests are executed in the CI. If you want, you can also add this in a follow-up PR.